src/ontology/mondo-edit.obo

```diff
@@ -117298,19 +117284,26 @@ id: MONDO:0006196
 name: endometrial serous adenocarcinoma
 def: "A high grade, aggressive adenocarcinoma arising from the endometrium. It is characterized by the presence of complex papillary patterns with cellular budding. Atypical mitoses, necrosis, and psammoma bodies may be present. It is classified as type II endometrial carcinoma and it is not associated with endometrial hyperplasia. It tends to invade deeply into the myometrium and spreads into the lymphatic vessels. Patients frequently present with spread of the tumor beyond the uterus at the time of diagnosis. The prognosis is usually poor." [NCIT:C27838]
 subset: otar {source="MONDO:OTAR"}
+synonym: "body of uterus serous adenocarcinoma" EXACT [MONDO:patterns/location]
 synonym: "endometrial serous adenocarcinoma" EXACT [NCIT:C27838]
 synonym: "serous endometrial adenocarcinoma" EXACT [NCIT:C27838]
-synonym: "uterine corpus serous adenocarcinoma" RELATED [NCIT:C27838]
-synonym: "uterine papillary serous carcinoma" EXACT [NCIT:C27838]
-synonym: "uterine serous adenocarcinoma" BROAD [NCIT:C27838]
+synonym: "uterine corpus serous adenocarcinoma" EXACT [NCIT:C27838]
+synonym: "uterine papillary serous carcinoma" EXACT [DOID:5750, NCIT:C27838]
+synonym: "uterine serous adenocarcinoma" BROAD [DOID:5750, NCIT:C27838]
 synonym: "uterine serous carcinoma" BROAD [NCIT:C27838]
```

Comment:
should the source `DOID:5750` be carried over from the [obsoleted term](https://github.com/monarch-initiative/mondo/pull/7647/files?diff=unified&w=0#diff-889e0a9ba62d62cc6ee7c5c9bead0075444d506253590ef0dcf4786904ac6768L63502) for this synonym?